api/porchconfig/v1alpha1/types.go

```diff
@@ -61,9 +53,6 @@ type RepositorySpec struct {
 	Deployment bool `json:"deployment,omitempty"`
 	// Type of the repository (i.e. git, OCI)
 	Type RepositoryType `json:"type,omitempty"`
-	// Content stored in the repository (i.e. Function, Package - the literal values correspond to the API resource names).
-	// TODO: support repository with mixed content?
-	Content RepositoryContent `json:"content,omitempty"`
```

Comment:
Isn't this a backward incompatible change? What happens in cases when a repository is pushed in with a RepositoryContent: Package ?

Reply:
Good point. I can revert. Or is there a cleaner alternative?
What is the k8s procedure for deprecation of an api attribute?

Reply:
Won't the "Content" field just be ignored?

Reply:
Based on the OpenAPI spec in the CRD, it will be just ignored. Would be good to check manually at least.

Reply:
Done